src/coreclr/src/.nuget/packaging.props

```diff
@@ -1,17 +1,13 @@
-<?xml version="1.0" encoding="utf-8"?>
 <Project>
 
   <PropertyGroup>
-    <PackageDescriptionFile>$(MSBuildThisFileDirectory)/descriptions.json</PackageDescriptionFile>
-    <PackageLicenseFile>$(CoreclrDir)/LICENSE.TXT</PackageLicenseFile>
-    <PackageThirdPartyNoticesFile>$(CoreclrDir)/THIRD-PARTY-NOTICES.TXT</PackageThirdPartyNoticesFile>
-
+    <PackageDescriptionFile>$(MSBuildThisFileDirectory)descriptions.json</PackageDescriptionFile>
+    <PackageLicenseFile>$(RepoRoot)LICENSE.TXT</PackageLicenseFile>
```

Comment:
Could `PackageThirdPartyNoticesFile` and `PackageLicenseFile` be shared with libraries packaging?